packages/usa-in-page-navigation/src/styles/_usa-in-page-navigation.scss

```diff
@@ -0,0 +1,121 @@
+@use "uswds-core" as *;
+@use "sass:color";
+
+.usa-in-page-nav-container {
+  align-items: flex-start;
+  display: flex;
+
+  // Set main landmark max width to ensure in-page nav will fit
+  main,
+  [class^="main"] {
+    max-width: units($theme-in-page-nav-main-content-max-width);
+    width: 100%;
+  }
+
+  .usa-in-page-nav__heading {
+    // TODO: This could be a setting?
+    @include typeset("ui", "2xs", 3);
+    @include u-font-weight("bold");
+    margin-bottom: units(2);
+    margin-top: 0;
+    padding: 0;
+  }
+
+  .usa-in-page-nav {
+    @include set-text-from-bg(
+      $bg-color: $theme-in-page-nav-background-color,
+      $context: "In-page navigation text color"
+    );
+    display: none;
+    margin-left: units($theme-in-page-nav-margin-left);
+    margin-top: units($theme-in-page-nav-margin-top);
+    max-width: units("card-lg");
+    order: 2;
+    position: sticky;
+    top: units($theme-in-page-nav-top);
+
+    @include at-media("tablet") {
+      display: block;
+      width: 100%;
+    }
+
+    nav {
```

Comment:
**Optional**, but I wish we used a class here. Even something like `.usa-in-page-nav__inner`. We've used a similar naming convention in banner, card, and header wrappers.

Reply:
`usa-in-page-nav__nav` seems reasonable here